app/controllers/products_controller.rb

```diff
@@ -24,6 +24,7 @@ def create
         @product = Product.new(product_params)
         respond_to do |format|
             if @product.save
+                @product << 
```

Comment:
This is weird?

Reply:
Yeaaah, I missed that one out. I'll delete it.